### DISCUSSION TITLE
Should I import the metadata ? after sync [Apr 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1666] [Discord Threads]
> 🙋‍♂️ **Kratos**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### not necessary  
#  
> 💁 **o1lab**  
#### if its all synced  
#  
> 🙋‍♂️ **Kratos**  
#### No what im trying to say is that its not synced because in nocoDB the said column is not a primary key and I cant edit the table.    
####    
#### In reality the column is a primary key which I checked.  
#  
> 💁 **o1lab**  
#### ok, like all tables created in nocodb have a primary key - how was that missed ?  
#  
> 💁 **o1lab**  
#### may be you can take a screenshot and paste it here  
#  
> 💁 **o1lab**  
#### of what you are seeing in a db client and nocodb  
#  
> 💁 **o1lab**  
#### + primary keys are not shown by default   
#### you would have to hover on fields and click 'show system fields'  
#  
> 🙋‍♂️ **Kratos**  
####   
![image0](https://cdn.discordapp.com/attachments/959812854570176572/960197563791835196/unknown.png)  
#  
> 🙋‍♂️ **Kratos**  
#### I can screen share if you want  
#  
> 🙋‍♂️ **Kratos**  
####   
![image0](https://cdn.discordapp.com/attachments/959812854570176572/960198224180838450/unknown.png)  
#  
> 🙋‍♂️ **Kratos**  
#### you see how token is a primary key but on nocoDB it does not show it as a primary key  
#  
> 🙋‍♂️ **Kratos**  
#### this is one of the other tables that has memberId as a primary key  
![image0](https://cdn.discordapp.com/attachments/959812854570176572/960198608865620068/unknown.png)  
#  
> 🙋‍♂️ **Kratos**  
#### Okay I fixed this issue  
#  
> 🙋‍♂️ **Kratos**  
#### You need to clear the metadata at the project list in the dashboard page and then open the project.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/959812854570176572)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
